fontbe/src/features.rs

```diff
@@ -111,22 +302,167 @@ fn write_debug_fea(context: &Context, is_error: bool, why: &str, fea_content: &s
     };
 }
 
+fn create_glyphmap(glyph_order: &IndexSet<GlyphName>) -> GlyphMap {
+    if glyph_order.is_empty() {
+        warn!("Glyph order is empty; feature compile improbable");
+    }
+    glyph_order
+        .iter()
+        .map(|n| Into::<FeaRsGlyphName>::into(n.as_str()))
+        .collect()
+}
+
+fn push_identifier(fea: &mut String, identifier: &KernParticipant) {
+    match identifier {
+        KernParticipant::Glyph(name) => fea.push_str(name.as_str()),
+        KernParticipant::Group(name) => {
+            fea.push_str("@");
+            fea.push_str(name.as_str());
+        }
+    }
+}
+
+/// Create a single variable fea describing the kerning for the entire variation space.
+///
+/// No merge baby! - [context](https://github.com/fonttools/fonttools/issues/3168#issuecomment-1608787520)
+///
+/// To match existing behavior, all kerns must have values for all locations for which any kerning is specified.
+/// See <https://github.com/fonttools/fonttools/issues/3168#issuecomment-1603631080> for more.
+/// Missing values are populated using the https://unifiedfontobject.org/versions/ufo3/kerning.plist/#kerning-value-lookup-algorithm.
+/// In future it is likely sparse kerning - blanks filled by interpolation - will be permitted.
+///
+/// * See <https://github.com/fonttools/fonttools/issues/3168> wrt sparse kerning.
+/// * See <https://github.com/adobe-type-tools/afdko/pull/1350> wrt variable fea.
+fn create_kerning_fea(kerning: &Kerning) -> Result<String, Error> {
+    // Every kern must be defined at these locations. For human readability lets order things consistently.
+    let kerned_locations: HashSet<_> = kerning.kerns.values().flat_map(|v| v.keys()).collect();
+    let mut kerned_locations: Vec<_> = kerned_locations.into_iter().collect();
+    kerned_locations.sort();
+
+    if log::log_enabled!(log::Level::Trace) {
+        trace!(
+            "The following {} locations have kerning:",
+            kerned_locations.len()
+        );
+        for pos in kerned_locations.iter() {
+            trace!("  {pos:?}");
+        }
+    }
+
+    // For any kern that is incompletely specified fill in the missing values using UFO kerning lookup
+    // Not 100% sure if this is correct for .glyphs but lets start there
```

Comment:
I would say yes, not least because fontmake has compiled .glyphs files by turning them into UFOs first for long time now and if Glyphs used a different kerning lookup algorithm than UFOs we would have known by now